source/_posts/2021-04-09-android-april-2021-release.markdown

```diff
@@ -0,0 +1,71 @@
+---
+title: "Home Assistant Companion Android App April 2021 Release"
+description: "Search/Filter, In-App Log Viewer & Shortcuts in the April 2021 Android Release"
+date: 2021-04-07 00:00:00
+date_formatted: "April 9, 2021"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2021-04-09-android-april-2021/Companion.png
+---
+
+Hey Everyone! Its time for the April 2021 Android release. We told you guys last month that we would be aligning our releases closer to Home Assistant core so here we are! Another month went by and the Android app has started to see more contributers coming along. Thank you for everyone who has come by this month to contribute to all the awesome features this month. Hopefully you will join us next month as we can always use more contributers to move forward faster.
```

Comment:
Additionally:

- `Its time` -> `It's time`
-  Doesn't flow: `Thank you for everyone who has come by this month to contribute to all the awesome features this month.`
- Missing comma: `Hopefully, you`